src/lib/Footer.svelte

```diff
@@ -1,6 +1,51 @@
+<script lang="ts">
+  //optional link to a page about the site/service
```

Comment:
Style nitpick: at the call-sites, saying `cookies={undefined}` should be unnecessary; just omit it. If there's a TS error, then set the default here ` = undefined`